resources/schema.gql

```diff
@@ -618,6 +659,55 @@ type FlowConfigurationBatching {
 
 union FlowConfigurationSchedule = TimeDelta | CronExpression
 
+type FlowConnection {
+	"""
+	A shorthand for `edges { node { ... } }`
+	"""
+	nodes: [Flow!]!
+	"""
+	Approximate number of total nodes
+	"""
+	totalCount: Int!
+	"""
+	Page information
+	"""
+	pageInfo: PageBasedInfo!
+	edges: [FlowEdge!]!
+}
+
+union FlowDescription = FlowDescriptionDatasetPollingIngest | FlowDescriptionDatasetPushIngest | FlowDescriptionDatasetExecuteQuery | FlowDescriptionDatasetCompaction | FlowDescriptionSystem
+
+type FlowDescriptionDatasetCompaction {
+	datasetId: DatasetID!
+	originalBlocksCount: Int!
+	resultingBlocksCount: Int
+}
+
+type FlowDescriptionDatasetExecuteQuery {
+	datasetId: DatasetID!
+	transformedRecordsCount: Int
+}
+
+type FlowDescriptionDatasetPollingIngest {
+	datasetId: DatasetID!
+	ingestedRecordsCount: Int
+}
+
+type FlowDescriptionDatasetPushIngest {
+	datasetId: DatasetID!
+	sourceName: String
+	inputRecordsCount: Int!
+	ingestedRecordsCount: Int
+}
+
+type FlowDescriptionSystem {
+	flowType: SystemFlowType!
+}
```

Comment:
This type looks different from `FlowDescriptionDataset*`. Should it be replaced with `FlowDescriptionSystemGC` so each system flow had its own description type, just like dataset flows do?

Reply:
Made similar flattened union. Just wanted to avoid dummy fields.